### PR TITLE
Wiki:dev: Fixed some typos

### PR DESCRIPTION
**Pull request by AsifkhanYT**

### CODE REVIEW

File: dev/source/docs/flightaxis.rst [hunk 1]
```diff
@@ -6,13 +6,13 @@ Understanding SITL with RealFlight
...
 
-- A model of the vehicle, including its physical characterisitcs and their impacts on the physics model, control mechanisms, and a graphical data base of its appearance.
+- A model of the vehicle, including its physical characterstics and their impacts on the physics model, control mechanisms, and a graphical data base of its appearance.
 
 - Physics models that will predict vehicle dynamics (attitude, velocity, positions,etc.) based on control inputs into the vehicle model
 
 - Graphics interpreter and display that takes the physics model's output and produces the graphics display of vehicle movement
 
-- A controller mechanism to produce control inputs into the physics model to command vehicle behavior
+- A controller mechanism to produce control inputs into the physics model to command vehicle behaviour
```
Comment (Hwurzburg) [inline dev/source/docs/flightaxis.rst:15]: behavior is correct spelling.....we generally use American English in the wiki

File: dev/source/docs/flightaxis.rst [hunk 2]
```diff
@@ -6,13 +6,13 @@ Understanding SITL with RealFlight
 
 This document will explain how SITL communicates with RealFlight for graphical simulations. In order to do any simulation you need several elements:
 
-- A model of the vehicle, including its physical characterisitcs and their impacts on the physics model, control mechanisms, and a graphical data base of its appearance.
+- A model of the vehicle, including its physical characterstics and their impacts on the physics model, control mechanisms, and a graphical data base of its appearance.
```
Comment (Hwurzburg) [inline dev/source/docs/flightaxis.rst:9]: this is incorrect....originall spelling is correct

File: dev/source/docs/flightaxis.rst [hunk 3]
```diff
@@ -25,11 +25,11 @@ This is illustrated below:
...
+They are input into the RealFlight vehicle model which is described by its .RFX file(archive actually). Most models have a "software radio" which take these "TX channels" and convert them into "RX channels", like a receiver. This "software radio" can also create dual rates, adjust trim, throw and reverses, etc. These received channels then drive motors, servos, or flight controller/mixers setup in the model files which drive the vehicle's actuators.
 
 The Physics models takes these inputs, including the vehicles described physical characteristics from the model's files and predicts the vehicle's motion which is then graphically displayed.
 
-The pilot provides the "autopilot" function of inputing stick/switch control via the simulator controller to direct vehicle desired behavior.
+The pilot provides the "autopilot" function of inputting stick/switch control via the simulator controller to direct vehicle desired behaviour.
```
Comment (Hwurzburg) [inline dev/source/docs/flightaxis.rst:32]: again behavior is not misspelled

**Comment by AsifkhanYT** [comment]
Thank you sir for the suggestion.. I think it would be better if I close this PR

**Pull request closed without merging**